packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,153 @@
+default.target=.
+
+## all templates to apply on target deployment env
+nuxeo.template.includes=default,s3binaries,postgresql,explorer-sitemode,explorer-virtualadmin
+# TODO: check logs config with 'docker' or 'docker-json' template
+
+## explorer confs mapping
+
+org.nuxeo.apidoc.apidocAdmin.password=${env:ORG_NUXEO_APIDOC_APIDOCADMIN_PASSWORD}
+
+
+## default confs mapping
+
+nuxeo.data.dir=${env:NUXEO_DATA_DIR:nxserver/data}
+nuxeo.server.dir=${env:NUXEO_SERVER_DIR:nxserver}
+
+# Default IP address binding
+nuxeo.bind.address=${env:NUXEO_BIND_ADDRESS:0.0.0.0}
+
+# publicly visible URL
+nuxeo.url=${env:NUXEO_URL:http://localhost:8080/nuxeo}
```

Comment:
Having this parameter makes sense to me to be able to configure the public URL that can not be introspected from Docker (I think).